manifests/e/ExtremeTuxRacer/ExtremeTuxRacer/0.8.0.0/ExtremeTuxRacer.ExtremeTuxRacer.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: ExtremeTuxRacer.ExtremeTuxRacer
+PackageVersion: 0.8.0.0
+PackageLocale: en-US
+Publisher: Extreme TuxRacer
+PublisherSupportUrl: https://sourceforge.net/projects/extremetuxracer/support
+PackageName: The ExtremeTuxRacer team
```

uhoh someone got the typo flu🙄
```suggestion
Publisher: The ExtremeTuxRacer team
PublisherSupportUrl: https://sourceforge.net/projects/extremetuxracer/support
PackageName: Extreme TuxRacer
```

---

Hahaha, thanks for pointing out.